app/views/spaces/_utilities.html.erb

```diff
@@ -32,4 +32,7 @@
   </ul>
 
   <p><%= link_to t('utilities.new.link_to'), space.location(:new, child: :utility) %></p>
+
+  <%= check_box_tag(:enforce_ssl) %>
+  <%= label_tag(:enforce_ssl, "Enforce SSL") %>
```

Comment:
This probably wants to be in a [`form_with(model: space.location)`](https://guides.rubyonrails.org/form_helpers.html#binding-a-form-to-an-object) block.

Reply:
fixed in 14fd46eacada467268b924c3461c557ec1d140e7 but I can't get the checkbox to be right next to the label text 🤔

Reply:
Hmmm, you'll probably need to use the `checkbox` or `label` methods  block syntax to put the checkbox within the label; rather than treat them as siblings.

Something like this maybe?

```erb
<%= form.label :enforce_ssl do
  <%= form.checkbox :enforce_ssl %> Enforce SSL
<%- end %>

```

Reply:
fixed in https://github.com/zinc-collective/convene/pull/1484/commits/0343d527e40d6defe99afe9b0294c4e040cb5384 and added a Save button